boards/tcc/topst_vcp45/Kconfig.defconfig

```diff
@@ -0,0 +1,88 @@
+# Copyright 2024 Hounjoung Rim <hounjoung@tsnlab.com>
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_TOPST_VCP45
+
+config ARM_CUSTOM_INTERRUPT_CONTROLLER
+	default y
+
+config THREAD_MONITOR
+	default y
+
+config THREAD_STACK_INFO
+	default y
+
+config INIT_STACKS
+	default y
+
+config TRACING
+	default y
+
+config TRACING_OBJECT_TRACKING
+	default y
+
+config TRACING_SYSCALL
+	default y
+
+config TRACING_THREAD
+	default y
+
+config TRACING_WORK
+	default y
+
+config TRACING_ISR
+	default y
```

Comment:
so why do any of these need to be enabled? Include for all the options below too